content/en/docs/collector/configuration.md

```diff
@@ -404,6 +410,56 @@ exporters:
     endpoint: http://localhost:9411/api/v2/spans
 ```
 
+## Connectors
+
+A connector is both an exporter and receiver. It consumes data as an exporter in
+one pipeline and emits data as a receiver in another pipeline. It may consume
+and emit data of the same data type, or of different data types. A connector may
+generate and emit data to summarize the consumed data, or it may simply
+replicate or route data.
+
+The `connectors:` section is how connectors are configured.
+
+> Configuring a connectors does not enable it. Connectors are enabled via
+> pipelines within the [service](#service) section.
+
+One or more connectors may be configured. By default, no receivers are
+configured. A basic example of all available connectors is provided below.
```

Comment:
This is missing the `forward` connector. 

Is it really intended that this will be kept up to date with examples for _all_ connectors? If not, I'd suggest removing that word.

Reply:
I copied that sentence from the section for receivers not reading that "all" there, we probably should not have that at any place, thanks for noticing!